docs/assets/css/less/bootstrap/dist/js/npm.js

```diff
@@ -0,0 +1,13 @@
+// This file is autogenerated via the `commonjs` Grunt task. You can require() this file in a CommonJS environment.
+require('../../js/transition.js')
+require('../../js/alert.js')
+require('../../js/button.js')
+require('../../js/carousel.js')
+require('../../js/collapse.js')
+require('../../js/dropdown.js')
+require('../../js/modal.js')
+require('../../js/tooltip.js')
+require('../../js/popover.js')
+require('../../js/scrollspy.js')
+require('../../js/tab.js')
+require('../../js/affix.js')
```

Comment:
Are we expected to run some manual step to concat the JS code? Or are all these part of bootstrap and we're using the already-concatenated version?

Reply:
No js script from bootstrap was used, so I deleted everything. Nothing requires additional building.